search-commons/src/main/java/no/unit/nva/search2/resource/ResourceParameter.java

```diff
@@ -90,6 +90,8 @@
 public enum ResourceParameter implements ParameterKey {
     INVALID(ParameterKind.INVALID),
     // Parameters used for filtering
+    SCIENTIFIC_REPORT_PERIOD_SINCE(CUSTOM, FieldOperator.GREATER_THAN_OR_EQUAL_TO, SCIENTIFIC_INDEX_YEAR),
+    SCIENTIFIC_REPORT_PERIOD_BEFORE(CUSTOM, FieldOperator.LESS_THAN, SCIENTIFIC_INDEX_YEAR),
```

Comment:
You can use Number instead of CUSTOM, if not handled, CUSTOM will fall through to keyword, which works fine now, but might break in thre future.